DirectXTex/DirectXTexDDS.cpp

```diff
@@ -1974,11 +2063,7 @@ HRESULT DirectX::SaveToDDSMemory(
                 if (fastpath)
                 {
                     size_t pixsize = images[index].slicePitch;
-                    if (memcpy_s(pDestination, remaining, images[index].pixels, pixsize))
```

Comment:
GNUC's library doesn't implement Appendix K stuff like ``memcpy_s``. I already do plenty with the counts to guard against issues that would trip up ``memcpy_s`` anyhow.

These were some of the few cases where I ever checked the return value of ``memcpy_s``, so in most places just using ``memcpy`` directly without any other changed worked. Here of course, it fails even when it works. Again, there's really no failure case here so using ``mempcy`` is fine.